src/components/TaskComponents/index.ts

```diff
@@ -0,0 +1,6 @@
+import AllTasks from "./AllTasks";
```

Comment:
No need to import first. 
`export * from "./FIlename"`

Reply:
Oh yes, thank you for the feedback

Reply:
I resolved this as you said.